drivers/storage/portworx/util/util.go

```diff
@@ -1472,7 +1480,9 @@ func CountStorageNodes(
 				storageNodesCount++
 			} else {
 				if _, ok := k8sNodesStoragePodCouldRun[node.SchedulerNodeName]; ok {
```

Comment:
The reason we changed the logic to calculate PDB is because during upgrades, at some point there exists a node where portworx is being run, but the k8s api doesnt not return the k8s node corresponding to the px node. So this px node wasn't being considered in the PDB calculation. 
I think we should remove mapping the px node to k8s node . We can directly increment storagenode count if its the cluster domain

Reply:
Done, not checking if node is a k8s node and directly moving ahead with cluster domain checks.